source/quickguides/quick-guide-generic.rst

```diff
@@ -0,0 +1,248 @@
+#expand machine_name# quick guide for running e3sm_diags v2
+=========================================================================
+
+1. Installation
+-----------------------------------------------------------
+
+We will use the e3sm_unifed environment to install.
+
+Most of the E3SM analysis software is maintained with an Anaconda metapackage.
+If you have an account on #expand machine_name#,
+then to get all of the tools in the metapackage in your path,
+use the command and activation path below.
+
+    ::
+
+     source #expand activation_path#
+
+Additionally, below are the relevant paths and the web address to run diags and
+visualize the results.
+
+<obs_path>: ``#expand obs_path#``
+
+<test_data_path>: ``#expand test_data_path#``
+
+<html_path>: ``#expand html_path#``
+
+<web_address>: ``#expand web_address#``
+     
+
+
+2. Config and run
+--------------------------------------------------------
+
+Running the annual mean latitude-longitude contour set
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Copy and paste the below code into ``run_e3sm_diags.py`` using your favorite text editor.
+Adjust any options as you like.
+
+   **Tip:** Some of E3SM's analysis machines (**Cori, Compy, Acme1, Anvil**)
+   have web servers setup to host html results.
+   On #expand machine_name#, create the directory ``#expand html_path`` using your username.
+   Set ``results_dir`` to ``#expand html_path#lat_lon_demo``
+   in ``run_e3sm_diags.py`` below. Then, you can view results via a web browser here:
+   #expand web_address#lat_lon_demo
+
+
+    .. code:: python
+
+        import os
+        from acme_diags.parameter.core_parameter import CoreParameter
+        from acme_diags.run import runner
+
+        param = CoreParameter()
+
+        param.reference_data_path = '#expand obs_path#climatology/'
+        param.test_data_path = '#expand test_data_path#climatology/'
+        param.test_name = '20161118.beta0.FC5COSP.ne30_ne30.edison'
+        param.seasons = ["ANN"]   #all seasons ["ANN","DJF", "MAM", "JJA", "SON"] will run,if comment out"
+
+        prefix = '#expand html_path#doc_examples/'
+        param.results_dir = os.path.join(prefix, 'lat_lon_demo')
+        #param.multiprocessing = True
+        #param.num_workers = 32
+        
+        # Use below to run all core sets of diags:
+        #runner.sets_to_run = ['lat_lon','zonal_mean_xy', 'zonal_mean_2d', 'polar', 'cosp_histogram', 'meridional_mean_2d']
+        # Use below to run lat_lon map only:
+        runner.sets_to_run = ['lat_lon']
+        runner.run_diags([param])
+
+
+Run in serial with:
+
+    ::
+
+        python run_e3sm_diags.py
+
+The above run has the same results as :ref:`the parameter file linked here <cori-params-v1>`,
```

Comment:
source/quickguides/quick-guide-cori.rst has an anchor point .. _cori-params-v2: where it should refer to

Reply:
So, each quick-guide should have an anchor `..<machine_name>-params-v2:`? We shouldn't always refer to `cori`, right?

Reply:
I have a second thought, maybe we link to source/quickguides/old/quick-guide-cori.rst and put the anchor point there? Let's only point to `cori`. I don't think we need to do different machines, cause this is only for reference.

Reply:
I'm thinking not even linking to anything. Maybe just display the code block for the parameters file directly. I think it would be confusing to direct to v1 docs, and furthermore to a specific machine in v1 docs. 

Also this links to `all_sets.py` when it seems like the correct params file would be the `myparams.py` earlier in the cori v1 docs (https://e3sm-project.github.io/e3sm_diags/docs/html/quickguides/old/quick-guide-cori.html)

Reply:
I agree that it's a better solution to display the section of codes.